commands/types/map/map.go

```diff
@@ -65,17 +66,19 @@ func getMap(clientConfig *hazelcast.Config, mapName string) (result *hazelcast.M
 		clientConfig = &hazelcast.Config{}
 	}
 	clientConfig.Cluster.ConnectionStrategy.Retry.InitialBackoff = types.Duration(1 * time.Second)
-	hzcClient, err := hazelcast.StartNewClientWithConfig(ctx, *clientConfig)
+	configCopy := *clientConfig
```

Comment:
It's better to use `clientConfig.Clone()`.

Reply:
Thank you, fixed